app/views/videos/_video.html.erb

```diff
@@ -1,14 +1,18 @@
-<div id="<%= dom_id video %>">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
```

Comment:
Kindly move into application.js or layout file.

Reply:
resolved this issue .